projects/backend/fronts.ts

```diff
@@ -84,18 +84,36 @@ export const getCollection = async (
             const meta = fragment && (fragment.meta as ArticleFragmentRootMeta)
             const kicker = (meta && meta.customKicker) || article.kicker || '' // I'm not sure where else we should check for a kicker
             const headline = (meta && meta.headline) || article.headline
-            const imageURL = (meta && meta.imageSrc) || article.imageURL
 
-            return [
-                article.path,
-                {
-                    ...article,
-                    key: article.path,
-                    kicker,
-                    headline,
-                    imageURL,
-                },
-            ]
+            switch (article.type) {
+                case 'crossword':
+                    return [
+                        article.path,
+                        {
+                            ...article,
+                            key: article.path,
+                            headline,
+                            kicker,
+                            crossword: (article.crossword as unknown) as Crossword,
```

Comment:
If anyone has a nicer way of doing this I'd love to know. When going from the Thrift type to the type here we lose precision going from i64 to number which means Typescript doesn't like it.

Reply:
Where does this `i64` come from, there shouldn't be an i64 in any TS code, TS shouldn't know about it but obvs there might be something funny with the thrift generation. I can come look ...